projects/laji/src/app/+observation/search-query.service.ts

```diff
@@ -344,4 +345,37 @@ export class SearchQueryService implements SearchQueryInterface {
   public queryUpdate(data = {}): void {
     this.queryUpdatedSource.next(data);
   }
+
+  public static hasValue(value): boolean {
+    const type = typeof value;
+    return (type !== 'undefined' && (type === 'boolean' || type === 'number' || (value && value.length > 0)));
+  }
```

Comment:
Add typing for the `value` input, otherwise if this is used for an object, it will return false (I assume it never receives an object in it's use case)